app/controllers/application_controller.rb

```diff
@@ -36,4 +39,13 @@ def set_locale
   def user_time_zone(&block)
     Time.use_zone(current_user.time_zone, &block)
   end
+
+  def user_not_authorized
+    flash[:error] = "You are not authorized to perform this action."
```

Comment:
Yep, we should. But now I saw that this message is never being checked in any specs because it has not showed up yet. So first we need to make sure pundit is actually blocking by adding checks to the controller specs.